web/src/condense.ts

```diff
@@ -23,6 +24,14 @@ This library implements two related, similar concepts:
 
 */
 
+const state: {
```

Comment:
I renamed this from `narrow_state`, since it's just the state for this module.

Might have been better to just have two variables set using `let`...